### DISCUSSION TITLE
Bonsai closes without warning

### DISCUSSION THREAD

**Discussion by PiezaKake** [bonsai-rx/bonsai#1408] [Q&A]
Hi,

I am running a Bonsai program where I record from 2 cameras and 1 mic. One of the cameras records continuously at 25 fps, the other one records at 250 fps only when the mouse enters the frame. The video and audio files are save in the .avi (without compression) and .wav formats respectively. The program seems to be working fine for a bit then closes after 2-15 min without a warning. Is there a way to get a crash report? Any ideas what may be going on?

**Comment by bruno-f-cruz** [comment]
May you add the minimal workflow that reproduces the issue? thanks
